basic_bootloader/src/bootloader/mod.rs

```diff
@@ -283,7 +283,9 @@ impl<S: EthereumLikeTypes> BasicBootloader<S> {
                         gas_used: tx_processing_result.gas_used,
                         gas_refunded: tx_processing_result.gas_refunded,
                         #[cfg(feature = "report_native")]
-                        native_used: tx_processing_result.native_used,
+                        computational_native_used: tx_processing_result.computational_native_used,
+                        #[cfg(feature = "report_pubdata")]
```

Comment:
Why do we have it under feature? Do you expect that we don't need it in some cases?

Reply:
imo breaking server isn't a very big problem